### PR TITLE
Increase tests coverage even more

### PR DESCRIPTION
**Pull request by darkdarkdragon**

### CODE REVIEW

File: src/api/server/server.js [hunk 1]
```diff
@@ -28,7 +28,9 @@ function isConnected(): boolean {
 function getServerInfo(callback: (err: any, data: any) => void): void {
   this.remote.requestServerInfo((error, response) => {
     if (error) {
-      callback(new common.errors.RippledNetworkError(error.message));
+      const message = error && error.remote && error.remote.error_message ?
```
Comment (clark800) [inline src/api/server/server.js]: Could use `_.get(error, 'remote', 'error_message')`

File: test/fixtures/api/rippled/account-tx.js [hunk 2]
```diff
@@ -196,15 +198,37 @@ module.exports = function(request, options={}) {
...
     id: request.id,
     status: 'success',
     type: 'response',
     result: {
-      marker: request.marker === undefined ? 'ABC' : undefined,
+      marker: marker === undefined ? undefined : String(marker),
       transactions: [
         {
-          ledger_index: 348860,
+          ledger_index: 348860 - +marker,
```
Comment (clark800) [inline test/fixtures/api/rippled/account-tx.js]: We're going to disallow this soon, so better to change it now: https://github.com/eslint/eslint/issues/1621

File: test/api-test.js [hunk 3]
```diff
@@ -341,4 +490,55 @@ describe('RippleAPI', function() {
   it('getLedgerVersion', function() {
     assert.strictEqual(this.api.getLedgerVersion(), 8819951);
   });
+
+  it('ledget utils - compareTransactions', function() {
```
Comment (clark800) [inline test/api-test.js]: `ledget` -> `ledger` and same below

File: test/api-test.js [hunk 4]
```diff
@@ -341,4 +490,55 @@ describe('RippleAPI', function() {
...
+        callback(new Error(), null);
+      }
+    }
+    utils.getRecursive(getter, 10, (error) => {
+      assert(error instanceof Error);
+      done();
+    });
+  });
+
+  it('validator', function() {
```
Comment (clark800) [inline test/api-test.js]: Can you add a case to test a valid address & secret pair?

Comment (sublimator) [inline test/api-test.js]: On that topic: I noticed previously we've been validating that an address (used for tx_json.Account) matches the secret used for signing, but what about regular keys?

Comment (clark800) [inline test/api-test.js]: I'm not totally clear on how regular keys work. Is the issue that validation will fail if someone signs the transaction using the regular key? If that's the case, then maybe we should just get rid of that validation check?

Comment (darkdarkdragon) [inline test/api-test.js]: If I understand right, signing with regular keys doesn't work, so that validation check is ok.

Comment (clark800) [inline test/api-test.js]: This reference seems to suggest you can: https://ripple.com/knowledge_center/generating-and-setting-a-regular-key/

Comment (darkdarkdragon) [inline test/api-test.js]: It doesn't work in ripple-lib
https://github.com/ripple/ripple-lib/issues/243
https://github.com/ripple/ripple-lib/issues/226

Comment (darkdarkdragon) [inline test/api-test.js]: Actually, with little changes to [sign.js](https://github.com/ripple/ripple-lib/blob/develop/src/api/transaction/sign.js) I was able to sign transaction using regular key.

Comment (sublimator) [inline test/api-test.js]: If it doesn't work, it's just a bug that could/should be fixed.

**Comment by clark800** [comment]
LGTM

**Pull request merged**